src/index.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable  import/no-cycle, import/no-mutable-exports */
+import './mystyles.css';
+import { store } from './modules/local-storage.js';
+import { addTask } from './modules/task.js';
+import { update } from './modules/update.js';
+import { deleter } from './modules/delete.js';
+import { status, clearer } from './modules/check.js';
+import { dragStart, dragOver, drop } from './modules/dragmodule.js';
+
+const addButton = document.querySelector('.fa-arrow-right-to-bracket');
+export const todo = document.querySelector('.todo');
+export const abnormal = document.querySelector('.abnormal');
+export const domTasks = document.querySelector('.all-activities');
+const clear = document.querySelector('.clear');
+
+export let tasks = [];
+
+let desc; let checkBox;
+
+export const displayTasks = () => {
+  tasks.forEach((task, i) => {
+    const taskPane = document.createElement('div');
+    taskPane.className = 'task-pane';
+    taskPane.setAttribute('draggable', 'true');
+    taskPane.setAttribute('id', `${task.index}`);
+
+    const topLeft = document.createElement('div');
+    topLeft.className = 'top-left';
+    topLeft.setAttribute('id', `${task.index}`);
+
+    const topRight = document.createElement('div');
+    topRight.className = 'top-right';
+    topRight.setAttribute('id', `${task.index}`);
+
+    const leftRight = document.createElement('div');
+    leftRight.className = 'left-right';
+    leftRight.setAttribute('id', `${task.index}`);
+
+    if (i % 2 === 0) leftRight.classList.add('dark');
+
+    const check = document.createElement('input');
+    check.className = 'check';
+    check.setAttribute('type', 'checkbox');
+    check.setAttribute('id', `${task.index}`);
+
+    const description = document.createElement('p');
+    description.textContent = task.description;
+    description.className = 'description';
+    description.setAttribute('id', `${task.index}`);
+    description.setAttribute('contenteditable', 'true');
+
+    const trash = document.createElement('i');
+    trash.className = 'fa-solid fa-trash-can fa-beat';
+
+    const anchor = document.createElement('i');
+    anchor.className = 'fa-solid fa-ellipsis-vertical';
+    anchor.setAttribute('id', `${task.index}`);
+
+    topLeft.append(check, description, trash);
+    topRight.appendChild(anchor);
+    leftRight.append(topLeft, topRight);
+
+    const ruler = document.createElement('hr');
+
+    taskPane.append(leftRight, ruler);
+
+    domTasks.appendChild(taskPane);
+
+    store();
+
+    topLeft.addEventListener('mouseenter', () => {
+      trash.style.display = 'block';
+      anchor.style.display = 'none';
+    });
+
+    topLeft.addEventListener('mouseleave', () => {
+      trash.style.display = 'none';
+      anchor.style.display = 'block';
+    });
```

Comment:
I think it would be better if you follow the DRY principle here. You could put the display codes in a function and call it in the event listeners

Reply:
Please @seyioyemade , I would need an explanation here because these code snippets are already in a function that started on line 20. 

Furthermore, I need an explanation on how this doesn't follow the DRY principle, especially because we are adding two different events to the same DOM element. Do you think there is another way to refactor this?